### PR TITLE
chore(docs):prometheus metrics documentation

### PR DESCRIPTION
**Pull request by amyshwang**
Capturing newly added metrics in #2536 and #2504

### CODE REVIEW

File: docs/third-party-tools/prometheus.md [hunk 1]
```diff
@@ -124,37 +132,49 @@ can be used to graph QuestDB-specific metrics which are all prefixed with
...
+| `questdb_json_queries_cached`            | gauge   | Number of current cached JASON queries.                                                                                                                                                                                                                 |
+| `questdb_pg_wire_select_queries_cached`  | gauge   | Number of current cached PGWire `SELECT` queries.                                                                                                                                                                                                        |
+| `questdb_pg_wire_update_queries_cached`  | gauge   | Number of current cached PGWire `UPDATE` queries.                                                                                                                                                                                                        |
```
Comment (amyshwang) [inline docs/third-party-tools/prometheus.md:177]: Newly added.

File: docs/third-party-tools/prometheus.md [hunk 2]
```diff
@@ -49,12 +52,17 @@ metrics.enabled=true
...
-docker run \
-  -e QDB_METRICS_ENABLED=TRUE \
-  -p 8812:8812 -p 9000:9000 -p 9003:9003 -p 9009:9009 \
-  questdb/questdb:6.1.3
-```
+<InterpolateReleaseData
+  renderText={(release) => (
+    <CodeBlock className="language-shell" title={"Docker"}>
+      {`docker run \\
+-e QDB_METRICS_ENABLED=TRUE \\
```
Comment (puzpuzpuz) [inline docs/third-party-tools/prometheus.md:59]: nit: would be nice to keep the original formatting:
```suggestion
  -e QDB_METRICS_ENABLED=TRUE \\
```

File: docs/third-party-tools/prometheus.md [hunk 3]
```diff
@@ -49,12 +52,17 @@ metrics.enabled=true
...
+-p 8812:8812 -p 9000:9000 -p 9003:9003 -p 9009:9009 \\
```
Comment (puzpuzpuz) [inline docs/third-party-tools/prometheus.md:60]:
```suggestion
  -p 8812:8812 -p 9000:9000 -p 9003:9003 -p 9009:9009 \\
```

File: docs/third-party-tools/prometheus.md [hunk 4]
```diff
@@ -49,12 +52,17 @@ metrics.enabled=true
...
+-v "$(pwd):/var/lib/questdb" \\
```
Comment (puzpuzpuz) [inline docs/third-party-tools/prometheus.md:61]:
```suggestion
  -v "$(pwd):/var/lib/questdb" \\
```

File: docs/third-party-tools/prometheus.md [hunk 5]
```diff
@@ -124,37 +132,49 @@ can be used to graph QuestDB-specific metrics which are all prefixed with
...
+| `questdb_memory_jvm_total`               | gauge   | Current size of Java memory heap in bytes.                                                                                                                                                                                                                    |
+| `questdb_memory_jvm_max`                 | gauge   | Maximum amount of Java heap memory that can be allocated in bytes.                                                                                                                                                                                            |
+| `questdb_json_queries_cached`            | gauge   | Number of current cached JASON queries.                                                                                                                                                                                                                 |
```
Comment (puzpuzpuz) [inline docs/third-party-tools/prometheus.md:175]:
```suggestion
| `questdb_json_queries_cached`            | gauge   | Number of current cached REST API queries.                                                                                                                                                                                                                 |
```

File: docs/third-party-tools/prometheus.md [hunk 6]
```diff
@@ -124,37 +132,49 @@ can be used to graph QuestDB-specific metrics which are all prefixed with
...
+| `questdb_jvm_minor_gc_count`             | counter | Number of times minor JVM garbage collection pause was triggered.                                                                                                                                                                                                          |
+| `questdb_jvm_minor_gc_time`              | counter | Total time spent on minor JVM garbage collection pauses in milliseconds.                                                                                                                                                                                      |
+| `questdb_jvm_unknown_gc_count`           | counter | Number of times unknown JVM garbage collection was triggered.                                                                                                                                                                                                               |
```
Comment (puzpuzpuz) [inline docs/third-party-tools/prometheus.md:149]:
```suggestion
| `questdb_jvm_unknown_gc_count`           | counter | Number of times JVM garbage collection of unknown type was triggered. Non-zero values of this metric may be observed only on some, non-mainstream JVM implementations.                                                                                                                                                                                                               |
```

File: docs/third-party-tools/prometheus.md [hunk 7]
```diff
@@ -124,37 +132,49 @@ can be used to graph QuestDB-specific metrics which are all prefixed with
...
+| `questdb_jvm_unknown_gc_time`            | counter | Total time spent on unknown JVM garbage collection in milliseconds.                                                                                                                                                                                           |
```
Comment (puzpuzpuz) [inline docs/third-party-tools/prometheus.md:150]:
```suggestion
| `questdb_jvm_unknown_gc_time`            | counter | Total time spent on JVM garbage collection of unknown type in milliseconds. Non-zero values of this metric may be observed only on some, non-mainstream JVM implementations.                                                                                                                                                                                           |
```

**Review by puzpuzpuz — APPROVED** [review]

**Pull request merged**